scipy/stats/tests/test_qmc.py

```diff
@@ -396,10 +394,56 @@ def test_subclassing_QMCEngine():
     assert_equal(sample_2, sample_2_test)
     assert engine.num_generated == 12
 
+
+def test_raises():
     # input validation
     with pytest.raises(ValueError, match=r"d must be an integer value"):
         RandomEngine((2,))  # noqa
 
+    msg = r"'u_bounds' and 'l_bounds' must be integers"
+    with pytest.raises(ValueError, match=msg):
+        engine = RandomEngine(1)
+        engine.integers(l_bounds=1, u_bounds=1.1)
+
+    msg = r"'l_bounds' and 'u_bounds' must be broadcastable"
+    with pytest.raises(ValueError, match=msg):
+        engine = RandomEngine(1)
+        engine.integers(l_bounds=[1, 2], u_bounds=4)
+
+
+def test_integers():
```

Comment:
These are fine, but I'd suggest adding the one obvious test that's missing: calling `integers` is essentially equivalent to calling `random`, `scale`ing, and `floor`ing, right? There could be a test for that.

Reply:
Done